demo.Dockerfile

```diff
@@ -4,4 +4,5 @@ WORKDIR /analyzer-lsp
 
 COPY rule-example.yaml /analyzer-lsp/rule-example.yaml
 COPY examples /analyzer-lsp/examples
+COPY examples/golang /analyzer-lsp/external-providers/golang-dependency-provider/golang-dependency-provider
```

Comment:
Is this intentional?  I think we should leave the examples under `/analyzer-lsp/examples`